modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -48,31 +50,36 @@
 }
 
 // Sidebar
-.sidebar-wrapper {
-	height: 100%;
-	width: 0;
 
-	&.open {
-		width: 320px;
-	}
-}
+$sidebarWidth: 320px;
 
-.sidebar-wrapper,
-.dashboard-content {
-	transition: 0.5s;
+.has-control-menu .sidebar {
+	top: 56px;
```

Comment:
If we are keeping this hard-coded and we are adding `$sidebarWidth`, we should add `$controlMenuHeight` as well (as you suggested in https://github.com/liferay-frontend/liferay-portal/pull/160#discussion_r460684604 but trying to keep it in the module)

Reply:
Or none of them https://github.com/liferay-frontend/liferay-portal/pull/164#discussion_r460881696 😁